grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -650,13 +648,12 @@ class ProductAvailabilityService {
         return data
     }
 
-    List getAvailableItems(Location location, List<Product> products) {
-        log.info("getQuantityOnHandByBinLocation: location=${location} product=${products}")
-        def data = []
+    List getAvailableItems(Location location, List<String> productsIds, boolean excludeNegativeQuantity = false) {
```

Comment:
@kchelstowski should it be `true` by defualt? Looks like we are very often doing `availableItems.findAll { it.quantityOnHand > 0 }`

Reply:
The only two places where this method is used are: `getAllAvailableBinLocations` and `getAvailableBinLocations` mentioned above. 
So I'd say we wouldn't achieve anything by changing that to `true`.